scripts/coe_player.lua

```diff
@@ -113,7 +113,6 @@ function Player.onPlayerCreated(event)
   local player = game.get_player(event.player_index)
   if Utils.getStartupSetting("coe_dev_mode") then setupForDevMode(player) end
   Utils.print("Player " .. player.name .. " created.", true)
-
```

Comment:
Blank line still snuck in.

Reply:
oh, I thought you wanted the blank line. :(

Reply:
yeah, I'm confused - my code does not have a blank line after the print.  Do you want one there?

Reply:
Meh its not hurting anything I guess it can stay